libs/dh/metering-point/feature-overview/src/graphql/get-metering-point-by-id.graphql

```diff
@@ -66,33 +86,11 @@ query GetMeteringPointById($meteringPointId: String!, $actorGln: String!) {
           name
           technicalContact {
             id
```

Comment:
question: is the `id` from `CustomerContact` going to override this `id`?

Reply:
Lets see